packages/core-data/src/controls.js

```diff
@@ -0,0 +1,17 @@
+export function regularFetch( url ) {
```

Comment:
what's regular fetch :D ?

Reply:
don't look :P 

I tracked this here #27391 basically we don't have a REST API endpoint to resolve a template, we use a hack (by fetching the actual frontend page with an extra flag)

Reply:
🙈 🙉 😃